daemon/control.c

```diff
@@ -542,6 +548,58 @@ control_handle_cmd_list_guestos_configs(UNUSED const ControllerToDaemon *msg, in
 	mem_free0(results);
 }
 
+/**
+ * Handles list_guestos_configs cmd.
```

Comment:
update comment to correct function name

Reply:
The function (including the comment) was removed after reusing the get_last_log functionality.